app/models/post.rb

```diff
@@ -0,0 +1,40 @@
+class Post < ActiveRecord::Base
+
+  attr_accessor :post_on_twitter
+  belongs_to :category
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :taggings
```

Comment:
What should happen with the taggings if the post is deleted? I would think its probably dependent: :destroy